scripts/globals/status.lua

```diff
@@ -2520,6 +2520,7 @@ xi.mobMod =
     NO_LINK             = 69, -- If set, mob cannot link until unset.
     NO_REST             = 70, -- Mob cannot regain hp (e.g. re-burrowing antlions during ENM).
     LEADER              = 71, -- Used for mobs that follow a defined "leader", such as Ul'xzomit mobs.
+    LEDGE_AGGRO      = 72, -- Used to increase vertical aggro range
```

Comment:
If you see just below I started an ASB mod range so we don't get conflicts with LSB